e2e/test/scripts/create_cluster.sh

```diff
@@ -5,54 +5,51 @@ set -o pipefail
 set -o nounset
 set -x
 
-export LINODE_API_TOKEN="$1"
+export LINODE_TOKEN="$1"
 export CLUSTER_NAME="$2"
-export K8S_VERSION="$3"
+export KUBERNETES_VERSION="$3"
+export CAPLI_VERSION="0.1.0"
```

Comment:
We have landed on CAPL for the naming instead of CAPLI, could we rename these references to CAPL?